Sources/App/configure.swift

```diff
@@ -16,21 +16,27 @@ private let jiraProjects = [
     "ETA"   : 17369, // Engagement Tribe - Activate
     "ETR"   : 17372, // Engagement Tribe - Retain
     "GW"    : 16949, // Triage UI
+    "HC"    : 17251, // HealthCheck (next-gen)
     // TODO: [IOSP-101/IOSP-147] Re-enable IDM once this board have migrated away from NextGen board
     // "IDM"   : 16903, // Identity Platform / Identity Management
```

Comment:
I think at that point we could as well remove these commented lines and mark IOSP-147 as "Won't Fix", because I think this board is obsolete anyway (replaced by `IDP`?)

Reply:
Done 👍